README.md

```diff
@@ -889,6 +891,53 @@ If these failures occur, add the warning to the list of permitted warnings found
 in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
 
 
+## macOS Big Sur: pyenv build failed
+
+[pyenv Big Sur GitHub issue](https://github.com/pyenv/pyenv/issues/1740)
+
+macOS Big Sur users may encounter a `build failed` error when installing python through pyenv. 
+To remedy this there is a patch file which runs the installer compatible with 
+macOS Big Sur (11.x).
+
+First is to install zlib and bzip2 using brew:
+```
+brew install zlib bzip2
+```
+
+`brew` output should tell you the steps to editing the path variables LDFLAGS and
+CPPFLAGS.
+The command can be placed in the `~/.bash_profile` to make the change persistent
+
+The command to install python 3.8.3 using the patch file is as follows:
+```
+pyenv install --patch 3.8.3 < <(curl -sSL https://github.com/python/cpython/commit/8ea6353.patch)
+```
+
+
+## macOS Big Sur: Replace preinstall LibreSSL
```

Comment:
Make this a required step when installing Python, merge it into the previous section.